src/ResourceManagement/RecoveryServices/Microsoft.Azure.Management.RecoveryServices/Generated/RecoveryServicesClient.cs

```diff
@@ -148,7 +145,7 @@ protected RecoveryServicesClient(System.Uri baseUri, System.Net.Http.HttpClientH
         /// <exception cref="System.ArgumentNullException">
         /// Thrown when a required parameter is null
         /// </exception>
-        internal RecoveryServicesClient(Microsoft.Rest.ServiceClientCredentials credentials, params System.Net.Http.DelegatingHandler[] handlers) : this(handlers)
+        public RecoveryServicesClient(Microsoft.Rest.ServiceClientCredentials credentials, params System.Net.Http.DelegatingHandler[] handlers) : this(handlers)
```

Comment:
Please do not change the constructor visibility modifiers in generatedcode.  Instead, since this is a partial class, please add the public constructors you want inside a Customizations folder in your library directory, or update the swagger spec to remove the directive to internalize constructors and update your generate.cmd accorsdingly.

It will likely be faster to add the constructors through customizations additions to the parital class.